rust/cymbal/src/fingerprinting/v1.rs

```diff
@@ -0,0 +1,109 @@
+use crate::{
+    error::Error,
+    types::{frames::Frame, Exception},
+};
+use reqwest::Url;
+use sha2::{Digest, Sha256};
+
+// Given resolved Frames vector and the original Exception, we can now generate a fingerprint for it
+pub fn generate_fingerprint(
+    exception: &Exception,
+    resolved_frames: Vec<Frame>,
```

Comment:
It's possible that not all frames are "resolved". We are going to add a boolean on the frame to denote whether it is or not